### DISCUSSION TITLE
Can we move blocks around?

### DISCUSSION THREAD

**Discussion by airallergy** [chanhx/crabviz#33] [Q&A] [answered]
First of all, thanks for creating this amazing extension! I wonder if we can move generated module blocks around, with the edges moved accordingly, or if there is a way to customise the location of blocks during graph generation?

**Comment by chanhx** [comment, marked as answer]
Thank you! I have thought about the dragging feature, and I think it's possible but not easy. I don't have a plan for this feature now due to other priorities, but any PRs for this are welcomed.

As for this,

> if there is a way to customise the location of blocks during graph generation?

No, we can't do that with Graphviz.
